extension/encoding/awslogsencodingextension/internal/unmarshaler/elb-access-log/fields.go

```diff
@@ -0,0 +1,10 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package elbaccesslogs
```

Comment:
This file has unit tests. Can you add some?

Reply:
This file only defines three constants. Are tests for these really useful? I don't see tests for the s3-access-logs [fields.go](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/encoding/awslogsencodingextension/internal/unmarshaler/s3-access-log/fields.go).

I could add some though, no problem.